frontend/src/components/SourcePage.js

```diff
@@ -46,8 +46,8 @@ class SourcePage extends Component {
               tabIndex={index + 1}
               className="line"
               role="button"
-              onClick={() => window.open( issueUrl( { id, gurmukhi } ) )}
-              onKeyPress={( ( { key } ) => key === 'Enter' && window.open( issueUrl( { id, gurmukhi } ) ) )}
+              onClick={() => window.open( issueUrl( { id, gurmukhi, page, source } ) )}
+              onKeyPress={( ( { key } ) => key === 'Enter' && window.open( issueUrl( { id, gurmukhi, page, source } ) ) )}
```

Comment:
`source` is actually a number here (source ID)... There are some additional props that the component has access to like `nameEnglish` etc which is the name of the source.

Reply:
Given that we're now doing `window.open` with the same params twice, we should refactor that into a function within the class